spring-boot-project/spring-boot-dependencies/build.gradle

```diff
@@ -1107,6 +1107,96 @@ bom {
 			]
 		}
 	}
+	library("Pulsar", "3.0.0") {
```

Comment:
Pulsar does not do a good job of dependency hygiene wrt what is server, client, shared and there a TON of published modules. Also, there is not one single arbiter in the repo that says "Here is what I publish" - its scattered about the repo in several different places and I am not clear which is which.

I instead got the list using https://github.com/mthmulders/mcs.

```
mcs search org.apache.pulsar -l 200 | grep ":3.0.0" | sed 's/3.0.0.*/3.0.0/' | sort | pbcopy
```
and
```
mcs search org.apache.pulsar -l 200 | grep ":0.3.0" | sed 's/0.3.0.*/0.3.0/' | pbcopy
```

Reply:
Yes, when you have to write a script to figure it out, its quite a lot of modules. I will file an issue and reach out to some committers but I am doubtful we will see any immediate changes in the SB 3.2 timeframe. I will probably end up submitting a PR for it (that would be faster - but have to find the time to do it).